tests/antithesis/test-template/robustness/traffic/main.go

```diff
@@ -159,10 +158,47 @@ func simulateTraffic(ctx context.Context, hosts []string, ids identity.Provider,
 		reports = append(reports, c.Report())
 		mux.Unlock()
 	}(compactClient)
+	dd := time.Duration(robustnessrand.RandRange(3, 37)) * traffic.DefaultCompactionPeriod
```

Comment:
`3-37` Where those number came from? 
The `DefaultCompactionPeriod` is 200ms, so what's the purpose of running it every 8s?

Reply:
With Jitter I don't think we should have any randomness here.